org.civicrm.bootstrapcivicrm/scss/civicrm/absences/_absences.scss

```diff
@@ -0,0 +1,58 @@
+.page-civicrm-absence-period,
```

Comment:
I wanted to keep everything part of **hrabsence** as when hrabsence is removed all the CSS would also be removed with that. 

So I used **hrabsence.css** whenever possible but all the pages(eg. New Public Holiday) are not using hrabsence.css hence I had to use custom-civicrm.scss for those pages. I created absences.scss so that we can remove that file when L&A is ready.